internal/cli/geoip/geoip.go

```diff
@@ -1,47 +1,33 @@
 package geoip
 
 import (
-	"fmt"
-
 	"github.com/alecthomas/kingpin"
 	"github.com/apex/log"
 	"github.com/ooni/probe-cli/internal/cli/root"
 	"github.com/ooni/probe-cli/internal/output"
-	"github.com/ooni/probe-cli/utils"
 )
 
 func init() {
 	cmd := root.Command("geoip", "Perform a geoip lookup")
 
-	shouldUpdate := cmd.Flag("update", "Update the geoip database").Bool()
-
 	cmd.Action(func(_ *kingpin.ParseContext) error {
 		output.SectionTitle("GeoIP lookup")
 		ctx, err := root.Init()
 		if err != nil {
 			return err
 		}
 
-		if err = ctx.MaybeDownloadDataFiles(); err != nil {
```

Comment:
How is the download of the data files now being handled?

Is it done as part of `MaybeLocationLookup`?

Reply:
Yes. You give the engine an existing directory. The engine makes sure there are assets in this directory matching the sha256 it knows. If they're missing or old, a new download starts.

In the common case, after checking the sha256, there's nothing else to do.